Bottom: 1edd1f9e4948e5c61ebfe2bc5e084d01576ccbae
Top:    038dde4ad199b4076c84930cf97e6347b9e2a2d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 22:17:13 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 727786a..2b90e70 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -295,6 +295,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
